bower.json

```diff
@@ -20,7 +20,6 @@
   ],
   "dependencies": {
     "requirejs": "~2.1.15",
-    "requirejs-text": "~2.0.12",
-    "mobify-velocity": "~1.1.2"
```

Comment:
Adding the `mobify-velocity` bower dependency to my `.bower.json` in my project root solved the issue.